pcsx2/GS/Renderers/HW/GSTextureReplacements.cpp

```diff
@@ -112,7 +112,7 @@ namespace GSTextureReplacements
 	static GSTextureCache::HashCacheKey HashCacheKeyFromTextureName(const TextureName& tn);
 	static std::optional<TextureName> ParseReplacementName(const std::string& filename);
 	static std::string GetGameTextureDirectory();
-	static std::string GetDumpFilename(const TextureName& name, u32 level);
+	std::string GetFilename(const TextureName& name, const std::string& subdirectory, u32 level);
```

Comment:
What was the point of changing this function?

Reply:
To reuse it for replacement texture existence. It is not needed now, I removed it.